### ISSUE TITLE
False positives for DL3008 and DL3015

### ISSUE THREAD

**Issue by patrickhoefler** [zabio3/godolint#40]
When I run `godolint` v1.0.1 on 

```
FROM ubuntu:jammy-20231004@sha256:2b7412e6465c3c7fc5bb21d3e6f1917c167358449fecac8176c6e496e5c1f05f

RUN \
  apt-get update \
  && apt-get install -y --no-install-recommends \
  fonts-dejavu="2.37-2build1" \
  graphviz="2.42.2-6" \
  && rm -rf /var/lib/apt/lists/*
```

I get:

```
#3 DL3008 Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`. 
#3 DL3015 Avoid additional packages by specifying `--no-install-recommends`.
```

As far as I can tell, both of these are false positives.
